### DISCUSSION TITLE
GIF Support

### DISCUSSION THREAD

**Discussion by ziiw** [outline/outline#4580] [Ideas]
Hey, I've been using Outline in my company and I would love to be able to use GIFs in my documents. Especially having them animated. Right now you can add GIFs, but they just don't play and stay at the first frame.

The behaviour:
- Finding a link of a GIFs on internet
- Pasting the link in the document
- Converting the URL in a playing GIF

Other option:
- Having a GIPHY integration in the / menu
- Search in the menu
- Add the GIF in the document

What do you think ? Thanks, Cheers

**Comment by Jatin-MYST** [comment]
Yes, I recently had the need to add gifs to our wiki.
But was a bit disappointed that outline doesn't support it.

This feature is a very useful in documentation.
Having to put 'click this then click that' makes the docs needlessly lengthy.
A gif can make explaining complex procedures more compact.

**Reply by tommoor** [reply]
We support gifs as any other image format - if you’re only seeing the first frame it’s because only the first frame has been copied to the clipboard.

You can get around this by uploading the image rather than pasting

**Reply by Jatin-MYST** [reply]
ahh.. yep. That did it. You were totally right.
I was right clicking copy on a gif from a website and trying to paste it.
Saving locally then uploading worked perfectly.
Thanks so much @tommoor
